backend/src/Altinn.Authorizationadmin/Altinn.AuthorizationAdmin.Core/Models/ResourceRegistry/ServiceResource.cs

```diff
@@ -10,86 +14,103 @@ public class ServiceResource
         /// <summary>
         /// The identifier of the resource
         /// </summary>
+        [JsonProperty]
         public string Identifier { get; set; }
 
         /// <summary>
         /// The title of service
         /// </summary>
+        [JsonProperty]
         public Dictionary<string, string> Title { get; set; }
 
         /// <summary>
         /// Description
         /// </summary>
+        [JsonProperty]
         public Dictionary<string, string> Description { get; set; }
 
         /// <summary>
         /// Description explaining the rights a recipient will receive if given access to the resource
         /// </summary>
+        [JsonProperty]
         public Dictionary<string, string> RightDescription { get; set;  }
 
         /// <summary>
         /// The homepage
         /// </summary>
-        public string Homepage { get; set; }    
+        [JsonProperty]
+        public string Homepage { get; set; }
 
         /// <summary>
         /// The status
         /// </summary>
+        [JsonProperty]
         public string Status { get; set; }
 
         /// <summary>
         /// When the resource is available from
         /// </summary>
-        public DateTime ValidFrom { get; set; } 
+        [JsonProperty]
+        public DateTime ValidFrom { get; set; }
 
         /// <summary>
         /// When the resource is available to
         /// </summary>
+        [JsonProperty]
         public DateTime ValidTo { get; set; }
 
         /// <summary>
         /// IsPartOf
         /// </summary>
+        [JsonProperty]
         public string IsPartOf { get; set; }
 
         /// <summary>
         /// IsPublicService
         /// </summary>
+        [JsonProperty]
         public bool IsPublicService { get; set; }
 
         /// <summary>
         /// ThematicArea
         /// </summary>
+        [JsonProperty]
         public string? ThematicArea { get; set; }
 
         /// <summary>
         /// ResourceReference
         /// </summary>
+        [JsonProperty]
         public ResourceReference? ResourceReference { get; set;  }
```

Comment:
ResourceReference property is being changed to a List of references in: 
https://github.com/Altinn/altinn-resource-registry/pull/29

Reply:
Also updates ReferenceType with two new values which must be added.